### DISCUSSION TITLE
GIFs with 100% width

### DISCUSSION THREAD

**Discussion by vinkla** [community/community#17088] [Accessibility]
It seems `<img src="..." width="100%" />` has stopped working since [this update](https://github.blog/changelog/2022-05-19-option-to-prevent-animated-images-from-playing-automatically/) (_probably_). The width is now ignored, is this the expected behaviour?

**Input:**

```
<img src="https://media.giphy.com/media/xiSZGYjpYgzkipNS7x/giphy.gif" width="100%">
```

**Output:**

<img src="https://media.giphy.com/media/xiSZGYjpYgzkipNS7x/giphy.gif" width="100%">

This has previously been discussed in here: https://github.com/github/feedback/discussions/16910#discussioncomment-2789016

**Comment by dipree** [comment]
Thanks for the note. We are indeed doing some magic when `width="100%"` is applied to images. Let me see how we can bring this back to GIFs. That said, `width="100%"` is not valid markup, `width` only accepts values, no units, it's always in pixels.

**Reply by vinkla** [reply]
Thanks! Is there any other way to make the images fill up the width of a Markdown file which is valid?

**Comment by dipree** [comment]
The problem with a percentage unit in the `width` attribute has been fixed.

**Reply by vinkla** [reply]
Great! When will it be available?

**Reply by dipree** [reply]
It should already be solved. Can you validate it?

**Reply by vinkla** [reply]
The GIF in the [start of the discussion thread](https://github.com/github/feedback/discussions/17088#discussion-4090376) isn't full width.

**Reply by dipree** [reply]
How about now?

**Reply by vinkla** [reply]
<img src="https://media.giphy.com/media/czubJ08i7deuKGJE9A/giphy-downsized.gif" width="100%" />

**Comment by cehfisher** [comment]
Hi @vinkla @dipree - this issue appears to be resolved. Could you please confirm and close the issue? Thank you! (cc @queenofcorgis)

**Reply by vinkla** [reply]
It has been fixed ✅

**Reply by cehfisher** [reply]
Awesome! Can you please close this issue as resolved @queenofcorgis - thanks!
